src/browser/appwindow.js

```diff
@@ -99,8 +109,11 @@ AppWindow.prototype.initYoProcess = function () {
   this.yoProcess.on('message', function (msg) {
     console.log('APP', msg);
 
-    this.sendCommandToBrowserWindow(msg.event, msg.data);
+    if (msg.event === 'generator:done') {
+      this.openProject(msg.data);
+    }
```

Comment:
I added another function in the new commit which will emit all incoming child events to Appwindow.